packages/neo-one-client-switch/src/node/LedgerTransport.ts

```diff
@@ -0,0 +1,12 @@
+import TransportNodeHid from '@ledgerhq/hw-transport-node-hid';
+import { Ledger } from '../common';
+
+const ledgerNodeTransport: Ledger = {
+  type: `ledger-node`,
```

Comment:
I would just keep these named the same, just `'ledger'` (also no backticks for plain strings). I forget what the implications are, but I think it's the right way to do it.